### DISCUSSION TITLE
add new provider

### DISCUSSION THREAD

**Discussion by command-z-z** [apache/airflow#29866] [General]
### Description

i have written a new provider,how can i contribute it?

### Use case/motivation

nothing

### Related issues

nothing

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
First of all please dont use issues for questions. When you create an issue there is a clear information that you should use discussions for questions, rather than creating issue (including the link).

Most likely you should not contribute provider to airflow - you can release it yourself and add it to  https://airflow.apache.org/ecosystem/  (click suggest a change on this page button at bottom-right to open PR).

We are pretty reluctant on accepting new providers to be managed by community - it must be something that is universally useful and ideally not connected to external services, that can usually manage the providers on their own.  There are certain expectations we have for quality and eligibility for our release process has to pass before the provider gets accepted by us in the community - we need to make sure we will be able to maintain it https://github.com/apache/airflow#release-process-for-providers 

I think if you REALLY want to contribute something to be provider you should start with describing your provider and why you think it is a good idea to the devlist (see https://airflow.apache.org/community/ for details on how to add it). You can also see some recent discussions which almost universally ended with the suggestion to release a provider by the one who can maintain them in the future:

Here are few examples but you can look up the devlist yourself

* https://lists.apache.org/thread/1gtw5vyypxh0p72wh4dss7cllcvhgh01
* https://lists.apache.org/thread/qk2co6trd7gm57744shprw2fhgmjr637
* https://lists.apache.org/thread/hvl2sg7mc6gwxs1h5kzhrcdtt8cc36dd
